### PR TITLE
Remove unused code

### PR DESCRIPTION
**Pull request by dims**
Change-Id: I7dfbad1c842f9484ede6572f3eb9c3aed5f7d613

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
:tada: 
/approve

File: test/images/no-snat-test-proxy/main.go [hunk 1]
```diff
@@ -18,14 +18,12 @@ package main
...
-	"net/http"
-	"os"
-	"strings"
-
 	"github.com/spf13/pflag"
+	"io/ioutil"
 	cliflag "k8s.io/component-base/cli/flag"
 	"k8s.io/component-base/logs"
+	"net/http"
+	"os"
```
Comment (liggitt) [inline test/images/no-snat-test-proxy/main.go]: stdlib imports got mixed together here

File: third_party/forked/golang/template/funcs.go [hunk 2]
```diff
@@ -75,14 +75,6 @@ func addValueFuncs(out map[string]reflect.Value, in FuncMap) {
 	}
 }
```
Comment (liggitt) [inline third_party/forked/golang/template/funcs.go]: prefer leaving the code in third_party/forked alone for things like this

Comment (dims) [reply third_party/forked/golang/template/funcs.go]: Fixing

File: test/images/no-snat-test-proxy/main.go (hunk 1 above)
Comment (dims) [reply test/images/no-snat-test-proxy/main.go]: oops!

File: third_party/forked/gonum/graph/simple/simple.go [hunk 3]
```diff
@@ -7,8 +7,6 @@
 package simple
 
 import (
-	"math"
```
Comment (liggitt) [inline third_party/forked/gonum/graph/simple/simple.go]: same comment about third_party

**Pull request merged**